server.js

```diff
@@ -1,15 +1,191 @@
+//Imports dotenv into file
 require("dotenv").config();
+//Imports express into file
 const express = require("express");
 const path = require("path");
+//Imports mongoose into file
+const mongoose = require("mongoose");
+//Sets up connection to the database with mongoose
+const db = mongoose.connection;
 
+//Creates a variable to initialize port
 const port = process.env.PORT || 8000;
+//Express server setup
 const app = express();
 const staticDir = process.env.DEV ? "./client/public" : "./client/build";
 
+//Sets up a connection to MongoDB
+const url = `mongodb+srv://${process.env.user}:${process.env.password}@react-chat-dillon-jo.nfgg1.mongodb.net/react-chat`;
+
+//Connects Mongoose to the MongoDB database
+mongoose.connect(url, { useNewUrlParser: true, useUnifiedTopology: true });
+
+//Sets up a static directory
 app.use(express.static(staticDir));
+//Sets up express middleware
+app.use(express.urlencoded({ extended: true }));
+
+//Sets up schema for chats
+const chatSchema = new mongoose.Schema({
+  username: { type: String, required: true },
+  message: { type: String, required: true },
```

Comment:
When you type a really long message it breaks your formatting, you could add another property to the message/username to stop people from entering really long messages to combat this. Schema looks great though 👍